terraform/modules/k8s-addons/eks-external-secrets.tf

```diff
@@ -154,4 +154,5 @@ resource "helm_release" "external_secrets" {
     local.external_secrets_values
   ]
 
+  depends_on = [helm_release.aws_loadbalancer_controller]
```

Comment:
Why do we need this?

Reply:
Because during  `terragrunt apply` I get next error:
```
│ Error: 1 error occurred:
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│ 
│ 
│ 
│   with helm_release.external_secrets[0],
│   on eks-external-secrets.tf line 143, in resource "helm_release" "external_secrets":
│  143: resource "helm_release" "external_secrets" {
│ 
╵
╷
│ Error: 3 errors occurred:
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│ 
│ 
│ 
│   with helm_release.ingress_nginx[0],
│   on eks-ingress-nginx-controller.tf line 202, in resource "helm_release" "ingress_nginx":
│  202: resource "helm_release" "ingress_nginx" {
│ 
╵
╷
│ Error: 4 errors occurred:
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│       * Internal error occurred: failed calling webhook "mservice.elbv2.k8s.aws": failed to call webhook: Post "https://aws-load-balancer-controller-webhook-service.aws-load-balancer-controller.svc:443/mutate-v1-service?timeout=10s": no endpoints available for service "aws-load-balancer-controller-webhook-service"
│ 
│ 
│ 
│   with helm_release.loki_stack[0],
│   on eks-loki-stack.tf line 169, in resource "helm_release" "loki_stack":
│  169: resource "helm_release" "loki_stack" {
│ 
```
After added these depends_on error has vanished. I deploy k8s-addons with these depens_on twice, and everything was okay (without errors).